src/bulk/common.ts

```diff
@@ -214,6 +216,21 @@ export class BulkWriteResult {
     return this.result.ok;
   }
 
+  /** Returns document_ids that were actually inserted
+   * @internal
+   */
+  getCleanedInsertedIds(bulkResult: BulkResult, isOrdered: boolean) {
```

Comment:
```suggestion
  getSuccessfullyInsertedIds(bulkResult: BulkResult, isOrdered: boolean): Document[] {
```

"cleaned inserted ids" doesn't indicate what the method is doing whereas `getSuccessfullyInsertedIds` indicates that it's producing a list of successfully inserted ids

also - it's good practice to explicitly type the parameters and return types on methods.  you can rely on type inference inside functions but this prevents us from accidentally changing the return type of a function

Reply:
Sounds good, I'll implement this in my next push.